Web API's/WebAPI/StudentProjectMVC/Controllers/StudentController.cs

```diff
@@ -0,0 +1,72 @@
+﻿using BusinessLogic;
+using DataLayer;
```

Comment:
why using data layer into WebAPI layer?

Reply:
As our repository is using object of context class in a constructor for database manipulation, that is why I am Importing DataLayer.